### ISSUE TITLE
Setting concurrency>1 with single release breaks sync

### ISSUE THREAD

**Issue by mhulscher** [roboll/helmfile#287]
Executing helmfile sync with concurrency == x where x > 1 when a single release is defined will cause helmfile to install/upgrade this same release x amount of times (at the same time).

```
helmfile --version
helmfile version v0.31.0
```

Invocation
```
helmfile --log-level=DEBUG --file=stages/bootstrap-apps/helmfile.yaml sync --concurrency=3

...

Upgrading stable/nginx-ingress
exec: helm upgrade --install --reset-values ingress-nginx stable/nginx-ingress --version 0.28.1 --wait --force --namespace ingress --values /tmp/values233664952 --tiller-namespace=tiller
Upgrading stable/nginx-ingress
exec: helm upgrade --install --reset-values ingress-nginx stable/nginx-ingress --version 0.28.1 --wait --force --namespace ingress --values /tmp/values233664952 --tiller-namespace=tiller
Upgrading stable/nginx-ingress
exec: helm upgrade --install --reset-values ingress-nginx stable/nginx-ingress --version 0.28.1 --wait --force --namespace ingress --values /tmp/values233664952 --tiller-namespace=tiller
Error: UPGRADE FAILED: release: "ingress-nginx.v9" already exists
```

`helmfile.yaml`
```
repositories:
- name: stable
  url: https://kubernetes-charts.storage.googleapis.com

helmDefaults:
  tillerNamespace: tiller
  wait: true
  force: true

releases:
- name: ingress-nginx
  namespace: ingress
  chart: stable/nginx-ingress
  version: 0.28.1
```

**Comment by mumoshu** [comment]
Ah, thanks for the report! I'll fix it asap.
2018年9月4日(火) 17:36 Mitch Hulscher <notifications@github.com>:

**Comment by mumoshu** [comment]
The fix has been released as part of v0.32.0. Thanks for the report!
